nyc_data/ppe/data_mappings.py

```diff
@@ -145,7 +145,6 @@ def to_objects(self):
             quantity=self.quantity,
             vendor=self.vendor,
             raw_data=self.raw_data,
-            data_source=DataType.EDC_PPE,
             order_type=OrderType.Purchase,
         )
         deliveries = []
```

Comment:
In below line, what is the purposed of `delivery_day_1` and `delivery_day_2`?

Reply:
It's just the structure of the sheet - there are up to two delivery dates per row